packages/react/src/components/MultiSelect/MultiSelect.js

```diff
@@ -193,6 +193,13 @@ const MultiSelect = React.forwardRef(function MultiSelect(
     }
   }
 
+  const onKeyDown = (e) => {
+    if (e.keyCode === 8 && !disabled) {
```

Comment:
Should we use the internal `keyboard` function here?

https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Search/Search.js#L153-L157

Reply:
Thanks for the suggestion @tw15egan Didn't realize we had this :D just updated!